pkgs/development/python-modules/wled/default.nix

```diff
@@ -36,10 +42,17 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  postPatch = ''
+    # Upstream doesn't set a version for the pyproject.toml
+    substituteInPlace pyproject.toml \
+      --replace "0.0.0" "${version}" \
+      --replace "--cov" ""
+  '';
```

Comment:
Can we convince the upstream to set the version?

Reply:
No

Reply:
Respectfully, I want to help clear up a potential misunderstanding. There is no copying involved with this repo, see: https://github.com/NixOS/nixpkgs/tree/master/pkgs%2Fdevelopment%2Fpython-modules%2Fwled

Those are only build instructions, so the copying happens on the user machine (or shared build infrastructure).

If that wasn't an actual concern, please ignore me.

Reply:
Please remove my packages from the repository. Fairly easy request that I hope you guys can respect.

Thanks for respecting the wishes of the author of the packaged source here.